docs/samples/index.md

```diff
@@ -12,7 +12,7 @@
 | Usage       | Python Notebook                       | [Analyzing structured / semi-structured data in batch](python/batch_processing.ipynb)                                                           |
 | Usage       | Python Notebook                       | [Getting the identified entity value using a custom Operator](python/getting_entity_values.ipynb)                                               |
 | Usage       | Python                                | [Using Flair as an external PII model](python/flair_recognizer.py)                                               |
-| Usage       | Python                                | [Using Transformers as an external PII model](python/transformers_recognizer.py)                                               |
+| Usage       | Python                                | [Using Transformers as an external PII model](python/transformers_recognizer)                                               |
```

Comment:
Please add a short readme to this folder. Otherwise, I'm not sure how it will render on MKDocs.

Reply:
Added documentation